### PR TITLE
Added Current Limits to TalonSRXs

### PR DESCRIPTION
**Pull request by andrewmartin5**
all except for intake

### CODE REVIEW

File: src/main/java/frc/robot/maps/FrancoisMap.java [hunk 1]
```diff
@@ -22,6 +23,7 @@
 import edu.wpi.first.wpilibj.AnalogTrigger;
 import edu.wpi.first.wpilibj.GyroBase;
 import edu.wpi.first.wpilibj.smartdashboard.SendableRegistry;
+import frc.robot.subsystems.ControlPanel;
```
Comment (bot190) [inline src/main/java/frc/robot/maps/FrancoisMap.java]: This seems misplaced, please remove.

File: src/main/java/frc/robot/maps/RobotMap.java [hunk 2]
```diff
@@ -14,9 +14,18 @@
 import com.chopshop166.chopshoplib.sensors.IEncoder;
 import com.chopshop166.chopshoplib.sensors.MockDigitalInput;
 import com.chopshop166.chopshoplib.sensors.MockEncoder;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 
 public class RobotMap {
 
+    // TODO check thes values
+    public void BAGCurrentLimit(TalonSRX talon) {
```
Comment (bot190) [inline src/main/java/frc/robot/maps/RobotMap.java]: May want to call this `setBagCurrentLimits`

**Review by bot190 — COMMENTED** [review]
Just a few small changes, and we'll definitely want to test these values to make sure they aren't problematic.

File: src/main/java/frc/robot/maps/RobotMap.java [hunk 3]
```diff
@@ -14,9 +14,18 @@
...
 import com.chopshop166.chopshoplib.sensors.MockDigitalInput;
 import com.chopshop166.chopshoplib.sensors.MockEncoder;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 
 public class RobotMap {
 
+    // TODO check thes values
+    public void setBAGCurrentLimit(TalonSRX talon) {
+        talon.configContinuousCurrentLimit(10, 0);
+        talon.configPeakCurrentLimit(15, 0);
```
Comment (bot190) [inline src/main/java/frc/robot/maps/RobotMap.java]: Lets make this 25 for now

File: src/main/java/frc/robot/maps/RobotMap.java [hunk 4]
```diff
@@ -14,9 +14,18 @@
 import com.chopshop166.chopshoplib.sensors.IEncoder;
 import com.chopshop166.chopshoplib.sensors.MockDigitalInput;
 import com.chopshop166.chopshoplib.sensors.MockEncoder;
+import com.ctre.phoenix.motorcontrol.can.TalonSRX;
 
 public class RobotMap {
 
+    // TODO check thes values
+    public void setBAGCurrentLimit(TalonSRX talon) {
+        talon.configContinuousCurrentLimit(10, 0);
```
Comment (bot190) [inline src/main/java/frc/robot/maps/RobotMap.java]: Probably increase this to 15.

**Review by bot190 — CHANGES_REQUESTED** [review]
A few more requests, sorry. I just want to make sure this doesn't impact us during normal play while still providing some safety.

**Comment by msoucy** [comment]
@andrewmartin5 can you merge master into this, fix the conflicts, and push?

**Comment by msoucy** [comment]
This PR could use some TLC. And a rebase. :)

**Review by bot190 — APPROVED** [review]

**Pull request merged**